frontend/src/components/GovBanner.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import clsx from 'classnames';
+import { makeStyles } from '@material-ui/core/styles';
+import { ExpandLess, ExpandMore } from '@material-ui/icons';
+import flagIcon from '../assets/us_flag_small.png';
+import govIcon from '../assets/icon-dot-gov.svg';
+import httpsIcon from '../assets/icon-https.svg';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    position: 'relative',
+    width: '100%',
+    backgroundColor: '#f0f0f0',
+    fontSize: '0.87rem',
+    display: 'flex'
+  },
+  inner: {
+    width: '100%',
+    display: 'flex',
+    flexFlow: 'row nowrap',
+    alignItems: 'center',
+    maxWidth: 1440,
+    margin: '0 auto',
+    padding: `0 24px`,
+    [theme.breakpoints.down('sm')]: {
+      padding: '0 8px'
+    }
+  },
+  flag: {
+    padding: `0 ${theme.spacing()}px`
+  },
+  textWrap: {
+    display: 'flex',
+    flexFlow: 'row wrap',
+    alignItems: 'center',
+    flexGrow: 1,
+    [theme.breakpoints.down('xs')]: {
+      padding: `5px 0`
+    }
+  },
+  text: {
+    flex: '0 1 auto',
+    marginRight: theme.spacing(),
+    [theme.breakpoints.down('xs')]: {
+      flex: '0 0 100%'
+    }
+  },
+  btn: {
+    border: 'none',
+    background: 'none',
+    padding: 0,
+    outline: 'none'
+  },
+  btnExpand: {
+    display: 'flex',
+    alignItems: 'center',
+    color: '#005ea2',
+    textDecoration: 'underline'
+  },
+  infoInner: {
+    maxWidth: 800,
+    margin: '0 auto',
+    lineHeight: 1.5,
+    flexFlow: 'row wrap',
+    padding: `${theme.spacing(3)}px 0`
+  },
+  info: {
+    display: 'flex',
+    flex: '1 1 50%',
+    padding: `${theme.spacing(2)}px 0`,
+    [theme.breakpoints.down('sm')]: {
+      flex: '0 0 100%'
+    }
+  },
+  infoIcon: {
+    minWidth: 40
+  },
+  infoText: {
+    '& p': {
+      margin: `0 ${theme.spacing(2)}px`
+    }
+  }
+}));
+
+export const GovBanner: React.FC = () => {
+  const classes = useStyles();
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <>
+      <div className={classes.root}>
+        <div className={classes.inner}>
+          <img src={flagIcon} alt="usa flag" className={classes.flag} />
+          <div className={classes.textWrap}>
+            <div className={classes.text}>
+              An official website of the United States government
```

Comment:
Can we make the GovBanner have the same colors, icons, etc. as uswds? In fact, I wonder if we can just use the same GovBanner that comes with uswds; not sure if we should change the styling of that at all.

GovBanner in this PR:

![image](https://user-images.githubusercontent.com/1689183/92180451-0bec9780-ee15-11ea-9d22-35205167ee73.png)

Regular GovBanner:

![image](https://user-images.githubusercontent.com/1689183/92180480-20c92b00-ee15-11ea-9cfd-7ae8ada19d02.png)

Reply:
I think it's worth getting this whole component written this way. If we're moving away from uswds, it feels bad to build and bundle the entire sass library for this one component. I'm fine with leaving it if the overhead is worth it. Are you saying the icons aren't there?

Reply:
Yeah, the icons weren't there. It appears to be fixed now, so it looks good!

Super minor, but do you think we should add a bit more padding to the banner now?

![image](https://user-images.githubusercontent.com/1689183/92191956-f4bca280-ee32-11ea-89e8-9948811f9ba3.png)

Reply:
Good call, I bumped the font size down slightly and added a little bit of padding